modules/manage/pages/manage-security/configure-server-certificates.adoc

```diff
@@ -305,35 +309,56 @@ sudo cp ./public/chain.pem /opt/couchbase/var/lib/couchbase/inbox/chain.pem
 sudo cp ./private/pkey.key /opt/couchbase/var/lib/couchbase/inbox/pkey.key
 ----
 
+. Change ownership of the private key to `couchbase`.
```

Comment:
Strictly speaking we don't need the private key to be owned by user couchbase, we only need it to be 'read' by used couchbase.
Also the same is true for node chain file and all files in inbox/CA.
We can say something like the following here:
"Make sure all certificate and private key files can be read by user 'couchbase'. It can be achieved by, for example, changing ownership of the files to 'couchbase' and setting 0600 permissions"

Reply:
I have changed accordingly.